app/build.gradle

```
@@ -210,6 +210,7 @@ dependencies {
     implementation 'androidx.compose.ui:ui-tooling-preview'
 
     implementation 'androidx.compose.material3:material3'
+    implementation 'androidx.compose.material:material-icons-extended-android:1.7.7'
```

```suggestion
    implementation 'androidx.compose.material:material-icons-extended-android'
```
No need to specify the version here. It's included in the Compose BOM